templates/auth_ui_settings.html

```diff
@@ -12,13 +12,201 @@
   Click <a href="/logout">here</a> to sign out.
 </p>
 
+<!-- Security -->
 <section class="pane">
-  <section class="settings-row settings-section">
+  <!-- The title and the description -->
+  <section class="settings-row settings-page-section-with-title-desc-action">
+    <h2 class="title primary-txt">
+      {{ template "settings-page-security-section-title" }}
+    </h2>
+    <p class="description secondary-txt">
+      {{ template "settings-page-security-section-description" }}
+    </p>
+  </section>
+
+  {{ range $.Authenticators }}
+  <!-- Primary Password -->
+  {{ if eq .Type "password" }}{{ if eq .Kind "primary" }}
+  <section class="settings-row settings-page-section-with-title-desc-action">
+    <p class="title primary-txt">
+      {{ template "settings-page-primary-password-title" }}
+    </p>
+    <p class="description secondary-txt">
+    <!-- FIXME(ui): format localized date time -->
+    {{ template "settings-page-primary-password-description" (makemap "time" "FIXME") }}
+    </p>
+    <a class="action" href="#">
+      {{ template "change-password-button-label" }}
+    </a>
+  </section>
+  {{ end }}{{ end }}
+
+  <!-- Primary OOB OTP -->
+  {{ if eq .Type "oob_otp" }}{{ if eq .Kind "primary" }}
+  <section class="settings-row settings-page-section-with-title-info-desc-action">
+    <p class="title primary-txt">
+      {{ if eq (index .Claims "https://authgear.com/claims/oob_otp/channel_type") "email" }}
+      {{ template "settings-page-primary-oob-otp-title--email" }}
+      {{ end }}
+      {{ if eq (index .Claims "https://authgear.com/claims/oob_otp/channel_type") "sms" }}
+      {{ template "settings-page-primary-oob-otp-title--sms" }}
+      {{ end }}
+    </p>
+    <p class="info secondary-txt">
+      {{ if eq (index .Claims "https://authgear.com/claims/oob_otp/channel_type") "email" }}
+      <i class="fas fa-envelope" aria-hidden="true"></i>
+      {{ index .Claims "https://authgear.com/claims/oob_otp/email" }}
+      {{ end }}
+      {{ if eq (index .Claims "https://authgear.com/claims/oob_otp/channel_type") "sms" }}
+      <i class="fas fa-phone" aria-hidden="true"></i>
+      {{ index .Claims "https://authgear.com/claims/oob_otp/phone" }}
+      {{ end }}
+    </p>
+    <p class="description secondary-txt">
+    <!-- FIXME(ui): format localized date time -->
+    {{ template "settings-page-primary-oob-otp-description" (makemap "time" "FIXME") }}
+    </p>
+    <a class="action" href="/settings/identity">
+      {{ template "details-button-label" }}
+    </a>
+  </section>
+  {{ end}}{{ end }}
+  {{ end }}
+
+  <!-- MFA title -->
+  <section class="settings-row settings-page-section-with-title">
```

Comment:
It seems it still show when no secondary authenticator is configured. We may want to hide it if the section is empty?